integration-tests/bats/dump.bats

```diff
@@ -160,6 +159,61 @@ teardown() {
     dolt sql -q "CREATE VIEW view2 AS SELECT y FROM b;"
     dolt sql -q "CREATE TRIGGER trigger2 AFTER INSERT ON a FOR EACH ROW INSERT INTO b VALUES (new.x * 2);"
     dolt sql -q "INSERT INTO a VALUES (2);"
+    dolt sql -q "CREATE PROCEDURE p1 (in x int) select x from dual"
```

Comment:
Need a test for a trigger that uses `precedes` or `follows` syntax:

https://dev.mysql.com/doc/refman/8.0/en/create-trigger.html